utils_nlp/models/transformers/common.py

```diff
@@ -134,7 +141,11 @@ def fine_tune(
             optimizer = AdamW(optimizer_grouped_parameters, lr=learning_rate, eps=adam_epsilon)
 
         if scheduler is None:
-            scheduler = WarmupLinearSchedule(optimizer, warmup_steps=warmup_steps, t_total=t_total)
+            if TRANSFORMERS_VERSION == '<2.2.0':
```

Comment:
Why don't we upgrade to 2.2 and remove this check? Does anything else break if we upgrade?

Reply:
We can upgrade to 2.2 and remove this check to use the new function name `linear_schedule_with_warmup`, but for those people who already created the environment without upgrade transformers package, they will have problem.

All those solutions should work anyway.

Reply:
I think it's just cleaner to upgrade. We can update our old envs easily.

Reply:
I updated the fix to only use the latest function. We will need to merge for the fix of the pakcage version as well.

Reply:
I still think we should remove the check, and use the latest version anyway. Imaging having to add a check for every new feature that breaks old code.

Reply:
@saidbleik The check has been removed. Which check did you mean?

Reply:
Great, that works. Thanks. 
I'll upgrade the package on the test machine.